grails-app/assets/javascripts/streama/translations/EN_us.js

```diff
@@ -34,14 +34,14 @@ angular.module('streama.translations').config(function ($translateProvider) {
 			OVERVIEW: 'Overview',
 			GENRE: 'Genre',
 			TRAILER: 'Trailer',
-			SEASON: 'Season',
+			SEASON: 'Season'
 		},
 
 		MESSAGES: {
 			SHARE_SOCKET: 'By creating a new session you will be redirected back to this player, but this time you will have a unique session ID in the url. Share this with your friends to have a syncronized watching experience with them!',
-			FILE_MISSING: 'There is a problem with this content. It seems you removed the associated video file from it.. Share this with your friends to have a syncronized watching experience with them!',
+			FILE_MISSING: 'There is a problem with this content. It seems you removed the associated video file from it.',
```

Comment:
Oh oops, I thought it was the message about the synchronized playback.. Haha you're right of course